firmware/README.md

```diff
@@ -1,3 +1,14 @@
+# Note about Chinese clone Nano (from US Amazon link)
+ - Download drivers
+ Drivers: directly from the chip manufacturer (Chinese Language site):
+WCH 340 product home page: http://www.wch.cn/product/CH340.html
+http://www.wch.cn/download/CH341SER_LINUX_ZIP.html (linux should operate fine without the driver)
+http://www.wch.cn/download/CH341SER_MAC_ZIP.html
```

Comment:
Please add the links using the markdown format for links. `[name](link)`

e.g. [Mac](http://www.wch.cn/download/CH341SER_MAC_ZIP.html)